### PR TITLE
Maintaining my own (currently) unused guns and respriting them

### PR DESCRIPTION
**Pull request by KathrinBailey**
## About The Pull Request
• Resprites most of the CFA armory.
• CFA Ruby is now 10mm Magnum (original TG 10mm).
• CFA Snub remains 4.6x30mm.
• CFA Rifle renamed and is now 7.62; it's considerably more powerful. New magazine sprite.
• CFA Wildcat remains 32acp but has a new sprite and magazine sprites with a slight fire rate buff.
• Neatens up code files for consistency.
• More rubber bullets are bouncy.
• Reskinnable magazines, Gandalf's thing, now works with AP and Incendiary rounds.
• See: https://docs.google.com/document/d/1CE5g8DM5Wt3iZrOWwzq2tKOtmmiHAxVtFaGA9yKUyEo/edit#

## Why It's Good For The Game
Massively improved spritework and balance to make these weapons relevant.
image
_Sprites in-game._
image
_New reskinable magazines._
image
_Old at the top, new at the bottom!_

## The new positions of the CFA weapons:
**CFA Snub**
• Fits in pockets.
• 17 round capacity.
• 20 damage bullets.
• Secondary weapon.

**CFA Ruby**
• Fits in bags.
• 8 round capacity.
• 40 damage bullets.
• Secondary weapon (barely).

**CFA Wildcat**
• Fits in bags.
• 30 round capacity.
• 15 damage bullets.
• Secondary weapon.

**CFA Rifle**
• Doesn't fit in bags.
• 10 round capacity.
• 60 damage bullets.
• Primary weapon.

## Future expansion plans for the CFA weapon line.
Another primary weapon, basically a real-life MP7, using 4.6x30mm too. 
A third primary weapon, making three primaries, three secondaries. Something like an M4. 5.56x45mm, high damage, 15 to 20 bullet capacity.
Alternatively to the M4, because that might be too overpowered, an energy gun that focuses on non-lethal takedowns.

### CODE REVIEW

File: modular_skyrat/modules/modular_weapons/code/modules/projectiles/guns/ballistic/pistol.dm [hunk 1]
```diff
@@ -8,51 +8,87 @@
...
+/obj/item/ammo_box/magazine/multi_sprite/cfa_snub/rubber
+	ammo_type = /obj/item/ammo_casing/c46x30mm/rubber
+	round_type = AMMO_TYPE_RUBBER
+
+/obj/item/ammo_box/magazine/multi_sprite/cfa_snub/incendiary
+	ammo_type = /obj/item/ammo_casing/c46x30mm/inc
+	round_type = AMMO_TYPE_INCENDIARY
+
+/obj/item/ammo_box/magazine/multi_sprite/cfa_snub/empty
+	start_empty = 1
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/modular_weapons/code/modules/projectiles/guns/ballistic/pistol.dm]:
```suggestion
	start_empty = TRUE
```

File: modular_skyrat/modules/modular_weapons/code/modules/projectiles/guns/ballistic/pistol.dm [hunk 2]
```diff
@@ -8,51 +8,87 @@
...
+	possible_types = list(AMMO_TYPE_LETHAL, AMMO_TYPE_AP, AMMO_TYPE_RUBBER, AMMO_TYPE_HOLLOWPOINT, AMMO_TYPE_INCENDIARY)
+	ammo_type = /obj/item/ammo_casing/c10mm
+	caliber = CALIBER_10MM
 	max_ammo = 12
-	multiple_sprites = 2
+	multiple_sprites = AMMO_BOX_FULL_EMPTY_BASIC
 
-/obj/item/ammo_box/magazine/m46x30/empty
+/obj/item/ammo_box/magazine/multi_sprite/cfa_ruby/empty
 	start_empty = 1
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/modular_weapons/code/modules/projectiles/guns/ballistic/pistol.dm]:
```suggestion
	start_empty = TRUE
```

File: modular_skyrat/modules/modular_weapons/code/modules/research/designs/autolathe_designs.dm [hunk 3]
```diff
@@ -101,9 +101,9 @@
 	category = list("hacked", "Security")
 
 /datum/design/c45_rubber
-	name = ".45 Rubber Bullet"
+	name = ".45 Bouncy Rubber Bullet"
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/modular_weapons/code/modules/research/designs/autolathe_designs.dm]:
```suggestion
	name = ".45 Bouncy Rubber Ball"
```

**Review by Gandalf2k15 — APPROVED** [review]

File: modular_skyrat/modules/modular_weapons/code/modules/research/designs/autolathe_designs.dm [hunk 4]
```diff
@@ -101,9 +101,9 @@
...
 	id = "c45_rubber"
 	build_type = AUTOLATHE
 	materials = list(/datum/material/iron = 2000)
-	build_path = /obj/item/ammo_casing/c45/rubber
```
Comment (KathrinBailey) [inline modular_skyrat/modules/modular_weapons/code/modules/research/designs/autolathe_designs.dm:108]: Urhhj what did I do here? Oops?

**Comment by KathrinBailey** [comment]
I need to put rubber 10mm into autolathe and any other bullets that aren't in there into there

**Comment by KathrinBailey** [comment]
Neat. Everything is now in order.

**Pull request merged**